CHANGES.md

```diff
@@ -12,6 +13,7 @@ Language grammar improvements:
 - enh(php) Add `mixed` to list of keywords (#2997) [Ayesh][]
 - enh(php) Add support binary, octal, hex and scientific numerals with underscore separator support (#2997) [Ayesh][]
 
+[highlightjs-hlsl]: https://github.com/stef-levesque/highlightjs-hlsl
```

Comment:
Typically it's your name, giving you credit (by name)... but I'm ok with this if you'd prefer it... let me know.

Reply:
Well I wonder if highlightjs-hlsl could be added to the `highlightjs` organization. Might be easier if other people want to contribute to it.

Reply:
https://github.com/highlightjs/highlightjs-hlsl

Reply:
Thanks! I'll move the repo and fix the links.

Reply:
ok, I've moved the repo and fixes the links. Let me know if there's anything missing, or that could be improved 🙂